src/main/webapp/components/MenuInput.test.tsx

```diff
@@ -3,22 +3,45 @@ import * as React from 'react';
 import userEvent from '@testing-library/user-event';
 import { AnimationToggleProvider } from '@splunk/react-ui/AnimationToggle';
 import MenuInput from './MenuInput';
-import { mockCustomMenu } from '../tests/helpers';
+import { mockCustomMenu, MockCustomRenderable } from '../tests/helpers';
 import { getUnifiedConfigs } from '../util/util';
 
 jest.mock('../util/util');
 
 // for further TS support
-// const getUnifiedConfigsMock = getUnifiedConfigs as jest.Mock;
-let mockCustomMenuInstance;
+const getUnifiedConfigsMock = getUnifiedConfigs as jest.Mock;
+let mockCustomMenuInstance: MockCustomRenderable;
 
 beforeEach(() => {
     mockCustomMenuInstance = mockCustomMenu().mockCustomMenuInstance;
 });
 
-function setup(inputs) {
+interface UnifiedConfig {
```

Comment:
i would extract that interface to some other file as it is about config that is used in many files so potentially can be used in multiple places